shared/copyq-completion

```diff
@@ -0,0 +1,106 @@
+# shellcheck shell=bash
+
+commands=(
+    show hide toggle menu exit disable enable clipboard selection paste copy
+    count select next previous add insert remove edit separator read write
+    action popup tab removetab renametab exporttab importtab config eval session
+    help
+)
+
+list_sessions() {
+    ss -l -x | sed -n '/\/\.copyq[^_]*_s /{s/.*\/\.copyq-\?\([^_]*\)_s .*/\1/;p}'
```

Comment:
I know I suggested this, but I think I missed the fact that CopyQ session name can contain `_`.

Here is fixed and simplified version:

```
ss -l -x | sed -nE 's#.*/\.copyq-?(\S*)_s .*#\1#p'
```